source/Generic/LibraryExporter/Views/LibraryExporterView.xaml

```diff
@@ -64,6 +64,8 @@
                       Content="{DynamicResource LOCRecentActivityLabel}" Margin="0,0,0,5" />
                 <CheckBox IsChecked="{Binding Settings.Settings.ExportSettings.Roms}"
                       Content="Roms" Margin="0,0,0,5" />
+                <CheckBox IsChecked="{Binding Settings.Settings.ExportSettings.CompletionStatus}"
+                      Content="CompletionStatus" Margin="0,0,0,5" />
```

Comment:
The Content should use the Playnite localization resource like the other properties

Reply:
Ok I will use LOCCompletionStatus.